opendevin/core/main.py

```diff
@@ -177,6 +177,13 @@ async def on_event(event: Event):
     AgentCls: Type[Agent] = Agent.get_cls(args.agent_cls)
     agent = AgentCls(llm=llm)
 
+    # if max budget per task is not sent on the command line, use the config value
+    max_budget_per_task = (
```

Comment:
what if the config has none, neither? 🤔

Reply:
Then it's none, like it is today, and the controller doesn't check budget.